src/world/Explosion.php

```diff
@@ -214,11 +246,34 @@ public function explodeB() : bool{
 				}
 				$this->world->setBlockAt($pos->x, $pos->y, $pos->z, $airBlock);
 			}
+
+			$fireBlock = VanillaBlocks::FIRE();
+
+			foreach($this->fireIgnitions as $ignition){
+				$firePos = $ignition->getPosition();
+				$x = (int) $firePos->x;
+				$y = (int) $firePos->y;
+				$z = (int) $firePos->z;
+
+				$toIgnite = $this->world->getBlockAt($x, $y, $z);
+
+				if($toIgnite->getTypeId() === BlockTypeIds::AIR &&
+					$toIgnite->getSide(Facing::UP)->isSolid()){
```

Comment:
Why do you use `UP` here instead of `DOWN`? Also, `isSolid()` isn't fire block support requirement. It requires the supporting block to provide `SupportType::FULL`